node/src/Commands.ts

```diff
@@ -494,6 +494,300 @@ export function createSetBit(
     ]);
 }
 
+/**
+ * Represents a signed or unsigned argument encoding for the {@link BaseClient.bitfield|bitfield} or
+ * {@link BaseClient.bitfieldReadOnly|bitfieldReadOnly} commands.
+ */
+export interface BitEncoding {
```

Comment:
Since interfaces and classes are being added for this case, this may warrant a separate file.

Reply:
I'll wait to get the teams input on your message, and move it to a separate file if thats what we decide

Reply:
in general, I'd prefer to move related types to a single file.  bit options are only going to be used by bit commands... and could go into a separate file from commands.ts. 
I'm not sure what that looks like... we might consider proposing to Avi

Reply:
Makes sense to me, since these commands only deal with some of the bit options I propose we merge this first, check with Avi, and then move all the bit options at once if we decide to go ahead